pontoon/teams/views.py

```diff
@@ -271,16 +271,16 @@ def request_item(request, locale=None):
         template = get_template("teams/email_request_item.jinja")
         mail_body = template.render(payload)
 
+        cc = list(locale.managers_group.user_set.values_list("email", flat=True))
+        if user.profile.contact_email:
+            cc = set(cc + [user.profile.contact_email])
```

Comment:
We should CC the user that has made the request even if their `contact_email` address is not set.